api/controllers/install/infra.go

```diff
@@ -22,23 +25,25 @@ func (c *InstallController) SetupInfra(ctx context.Context, ignorePreflightFailu
 
 	// Handle failed preflights
 	if preflightStatus.State == types.StateFailed {
-		// Report metrics for failed preflights
-		if c.metricsReporter != nil {
-			preflightOutput, err := c.GetHostPreflightOutput(ctx)
-			if err != nil {
-				return false, fmt.Errorf("get install host preflight output: %w", err)
-			}
-			if preflightOutput != nil {
-				c.metricsReporter.ReportPreflightsFailed(ctx, preflightOutput)
-			}
+		// Get preflight output for reporting
+		preflightOutput, err := c.GetHostPreflightOutput(ctx)
+		if err != nil {
+			return false, fmt.Errorf("get install host preflight output: %w", err)
 		}
 
 		// Check if we can proceed despite failures
 		if !ignorePreflightFailures || !c.allowIgnoreHostPreflights {
-			return false, fmt.Errorf("Preflight checks failed")
+			// Report metrics for failed preflights that are NOT bypassed
+			if c.metricsReporter != nil && preflightOutput != nil {
+				c.metricsReporter.ReportPreflightsFailed(ctx, preflightOutput)
+			}
```

Comment:
This is not needed as it should have [already](https://github.com/replicatedhq/embedded-cluster/blob/5c52173fa644edea3aa6dc4026063d5774d99e4e/api/internal/managers/preflight/hostpreflight.go#L141) been reported when they failed.